### ISSUE TITLE
Eclipse openHAB development IDE setup fails

### ISSUE THREAD

**Issue by hebs1** [openhab/openhab-distro#677]
Hi,
my IDE setup always fails and I have no idea why :-( I did it as described here:
https://docs.openhab.org/developers/development/ide.html

I selected "Eclipse IDE for Java Developers" and as project I only selected "openHAB Development":
![image](https://user-images.githubusercontent.com/37960314/38457131-c535d8b2-3a8c-11e8-9a1d-d1cd63d0b86c.png)

On installation I get this error "Cannot complete the request":
![image](https://user-images.githubusercontent.com/37960314/38457009-7fb7afd2-3a8b-11e8-9114-39cd208ccc79.png)

I also tried "Perform setup tasks" afterwards but issue is the same.

Completely new installation with all openHAB projects (Development + OH2AddOns + OH1AddOns + EclipseSmarthomeExtensions + ESH+OH2CoreBundles) also fail.

My Eclipse version is current Oxygen.3 Release (4.7.3)

Any idea?

**Comment by kaikreuzer** [comment]
No. I just tried it myself and all went smooth - so there is no general problem.
How does your IDE look like when you close the window with "Finish"? Are the projects in the workspace? Are errors shown?

**Comment by hebs1** [comment]
Thanks for your reply.
On every restart of eclipse I get same log:
![image](https://user-images.githubusercontent.com/37960314/38504845-1867fc24-3c16-11e8-9abc-c2e3ba2f949e.png)

These are the imported objects:
![image](https://user-images.githubusercontent.com/37960314/38504946-5482d49a-3c16-11e8-8a93-4c4f58a070d4.png)

If I run openHAB_Runtime I get this error:
![image](https://user-images.githubusercontent.com/37960314/38505055-8bb146f4-3c16-11e8-91fc-2c0d57e39f9e.png)
Corresponding log file:
![image](https://user-images.githubusercontent.com/37960314/38505130-bda9fe12-3c16-11e8-8cad-fc871b1e5bdc.png)

**Comment by hebs1** [comment]
I also tried an installation with Development and OH2 Addons: 
![image](https://user-images.githubusercontent.com/37960314/38508011-575dd770-3c1e-11e8-94ed-193bb8834033.png)
But same message „Cannot complete the request. Generating details“ at step „Target Platform openHAB Target Platform“:
![image](https://user-images.githubusercontent.com/37960314/38508024-60d9f37e-3c1e-11e8-86dc-c6ea328eb2ff.png)
Rest of installation was successful. These are the installed objects:
![image](https://user-images.githubusercontent.com/37960314/38508037-661f309c-3c1e-11e8-90b3-2e916cf0aa71.png)
But I have thousands of errors so openHAB_Runtime also doesn't run:
![image](https://user-images.githubusercontent.com/37960314/38508127-986e5870-3c1e-11e8-8f30-77bd1942eccf.png)

**Comment by hebs1** [comment]
Further information: I tried "Windows/Preferences/Plug-in Development/Target Platform" to reload openHAB Target Platform but I geht this errors:
![image](https://user-images.githubusercontent.com/37960314/38519728-19364572-3c41-11e8-8160-c6b809bec193.png)
And on click on "Edit" I get this information:
![image](https://user-images.githubusercontent.com/37960314/38519675-ead5978c-3c40-11e8-8132-5664f7281c24.png)

**Comment by kaikreuzer** [comment]
Are you behind some firewall? "Problems loading repositories" seems to indicate that you have some network issues...

**Comment by ppieczul** [comment]
My clean installation completes (Ubuntu) but IDE build fails everywhere - and there are also errors in target platform.

**Comment by hebs1** [comment]
I use standard Windows Firewall. I just deactivated it and retried it...but same issues.

**Comment by hebs1** [comment]
Oh my God...I don't really know why...but just now it worked!! I deleted "everything" in my C:\Users\User folder regarding Eclipse, and I also deactived and afterwards reactivated my Windows Firewall. Then IDE setup worked well (with some Firewall questions I accepted).
So next I try to do same installation including OH2 Addons. Will see if that also work ;-)

@Kai: thank you so much for your support!!!!
